velox/functions/sparksql/tests/ArithmeticTest.cpp

```diff
@@ -313,5 +313,17 @@ TEST_F(BinTest, bin) {
   EXPECT_EQ(bin(0), "0");
 }
 
+TEST_F(ArithmeticTest, hypot) {
+  const auto hypot = [&](std::optional<double> a, std::optional<double> b) {
+    return evaluateOnce<double>("hypot(c0, c1)", a, b);
+  };
+
+  EXPECT_EQ(hypot(3, 4), 5);
+  EXPECT_EQ(hypot(-3, -4), 5);
+  EXPECT_EQ(hypot(std::nullopt, std::nullopt), std::nullopt);
```

Comment:
Tests with nulls do not test the function, but the framework. Please, remove.

Let's add tests with floating point numbers as well.

Reply:
Updated. Thanks your suggession.